german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Du scheinst ja ziemlich DSA geschädigt zu sein.^^
Gambeson ist kein falscher Begriff vor allem wenn in der Vorlage schon Gambeson verwendet wurde. DSA Phobie hin oder her. Erster Google Eintrag ist dazu der Wikipedia Eintrag, in dem ja genau drin steht um was es sich handelt. Google mal Festes Wams und dann schau mal was da hilfreiches bei rum kommt. ;-)